### PR TITLE
implemenation: related list behavior

### PR DESCRIPTION
**Pull request by anerolabs**
Related - List Behavior
ticket: https://trello.com/c/EDkKmlDV/36-related-list-behavior

Related - List Behavior Styling
ticket: https://trello.com/c/T7Kflaa2/71-related-list-behavior-styling

Carousels working viably for now. Will like to clean up functionality a bit if time permits.

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -23,9 +23,13 @@ class App extends React.Component {
   updateAppProductId(productId) {
     this.setState({
       productId: productId
+    }, () => {
+      window.scrollTo({
```
Comment (lbc1013) [inline src/App.jsx:27]: I like the scrollTo method with this smooth option! 
I will use this for my review scroll button too.

File: src/components/related/RelatedProductsList.jsx [hunk 2]
```diff
@@ -44,21 +46,97 @@ class RelatedProductsList extends React.Component {
...
+              return <ProductCard key={i}
+                hidden={!indexesToShow.includes(i)}
+                currentList={currentList}
+                productId={productId}
+                productCardId={relatedId}
+                updateAppProductId={updateAppProductId} />
+            })}
+          </div>
+          <div className='related-arrow'>
+            <img className={showRightArrow ? 'related-right-arrow' : 'related-right-arrow hide'}
```
Comment (lbc1013) [inline src/components/related/RelatedProductsList.jsx:136]: this is great! I didn't know we're able to do the conditional rendering inside of className.

**Review by lbc1013 — APPROVED** [review]
looks good and I like how you calculate the width to see how many card can be displayed.

File: src/components/related/RelatedProductsList.jsx (hunk 2 above)
Comment (brynrmrzz) [reply src/components/related/RelatedProductsList.jsx:136]: 🧊 super cool

File: src/App.jsx (hunk 1 above)
Comment (brynrmrzz) [reply src/App.jsx:27]: Are you passing a callback into setState? 🤯

**Review by brynrmrzz — APPROVED** [review]

**Pull request merged**